selenium/che-selenium-core/src/main/java/org/eclipse/che/selenium/core/constant/Infrastructure.java

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright (c) 2012-2018 Red Hat, Inc.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *   Red Hat, Inc. - initial API and implementation
+ */
+package org.eclipse.che.selenium.core.constant;
+
+import com.google.inject.Singleton;
+
+@Singleton
+public enum Infrastructure {
+  docker,
+  openshift,
+  k8s,
+  supershift
```

Comment:
`supershift` is redundant here

Reply:
I like an idea of supporting `supershift` in Che :smile: